lib/manageiq/loggers/journald.rb

```diff
@@ -45,8 +45,11 @@ def add(severity, message = nil, progname = nil)
         # ActiveSupport::BroadcastLogger or the older ActiveSupport::Logger.broadcast
         # so we have to account for that difference when walking up the caller_locations
         # to get the "real" logging location.
-        callstack_start = ActiveSupport.gem_version >= Gem::Version.new("7.1.0") ? 7 : 3
-        caller_object = caller_locations(callstack_start, 1)&.first
+        caller_object = caller_locations.detect do |caller_loc|
+          next if caller_loc.path.end_with?("lib/active_support/broadcast_logger.rb", "logger.rb", "lib/active_support/logger.rb")
```

Comment:
"logger.rb" is a subset of "lib/active_support/logger.rb", so that last one is always ignored. Should we change it to:

```suggestion
          next if caller_loc.path.end_with?("lib/active_support/broadcast_logger.rb", "/logger.rb", "lib/active_support/logger.rb")
```

Reply:
Yeah so I had `lib/logger.rb` before, check out https://github.com/ManageIQ/manageiq-loggers/pull/92#discussion_r2103148881 for why that didn't work.  I agree I don't like `logger.rb` but the path seems to be kind of inconsistent

Reply:
Yeah, at a minimum then maybe this?  Only because all 3 end with logger.rb, so at that point that might as well be the only one :)

```suggestion
          next if caller_loc.path.end_with?("lib/active_support/broadcast_logger.rb", "/logger.rb", "lib/active_support/logger.rb")
```

Reply:
Yeah not great but better than going through the `$LOAD_PATH` every time :smile: